changelogs/unreleased/gh-7486-prohibit-roll-back-due-to-replication_synchro_timeout.md

```diff
@@ -0,0 +1,16 @@
+## feature/replication
```

Comment:
Regarding the documentation requests in both patches.

I would explicitly mention the new `replication.synchro_queue_max_size` and the `compat.box_cfg_replication_synchro_timeout` cluster configuration options to make it clear that the [configuration reference](https://www.tarantool.io/en/doc/latest/reference/configuration/configuration_reference/) needs an update.

Personally, I prefer to describe new options using the cluster configuration naming, because the original idea of this mechanism is to be the primary one for a user, while `box.cfg` is considered now as a low-level and compatibility configuration mechanism.

Reply:
> I would explicitly mention the new replication.synchro_queue_max_size and the compat.box_cfg_replication_synchro_timeout cluster configuration options to make it clear that the [configuration reference](https://www.tarantool.io/en/doc/latest/reference/configuration/configuration_reference/) needs an update.

Okay, I focused on this in the doc bot request.